typescript/sdk/package.json

```diff
@@ -14,10 +14,9 @@
     "@safe-global/safe-deployments": "1.37.8",
     "@solana/spl-token": "^0.3.8",
     "@solana/web3.js": "^1.78.0",
-    "@types/coingecko-api": "^1.0.10",
     "@wagmi/chains": "^1.8.0",
     "bignumber.js": "^9.1.1",
-    "coingecko-api": "^1.0.10",
+    "coingecko-api-v3": "^0.0.29",
```

Comment:
It's strange that they put out a new package instead of just making a new major version for the package they already had

Reply:
interesting, seems like there's no official JS/TS sdk for coingecko, so these are different authors